x/liquidstaking/keeper/tally_test.go

```diff
@@ -309,20 +328,20 @@ func (s *KeeperTestSuite) TestGetVotingPower() {
 	s.assertVotingPower(delD, sdk.ZeroInt(), delDbToken, sdk.ZeroInt())
 
 	// Test Farming Queued Staking of PoolTokens including bToken
-	s.farm(delC, sdk.NewInt64Coin(pool2.PoolCoinDenom, 500000000000))
-	s.assertVotingPower(delC, sdk.ZeroInt(), delCbToken, sdk.ZeroInt())
+	s.farm(delC, sdk.NewInt64Coin(liquidammtypes.ShareDenom(publicPosition1.Id), 5000000))
+	s.assertVotingPower(delC, sdk.ZeroInt(), delCbToken.SubRaw(4), sdk.ZeroInt())
```

Comment:
is there a reason for `SubRaw` was used?  Is it because of a decimal error?

Reply:
Yes, a small amount of error is caused by decimal truncation when calculating a position's underlying assets in x/amm.